repos/system_upgrade/common/actors/checkdnfpluginpath/libraries/checkdnfpluginpath.py

```diff
@@ -0,0 +1,32 @@
+from leapp import reporting
+from leapp.libraries.stdlib import api
+from leapp.models import DnfPluginPathDetected
+
+DNF_CONFIG_PATH = '/etc/dnf/dnf.conf'
+
+
+def check_dnf_pluginpath(dnf_pluginpath_detected):
+    """Create an inhibitor when pluginpath is detected in DNF configuration."""
+    if dnf_pluginpath_detected.is_pluginpath_detected:
+        reporting.create_report([
+            reporting.Title('Detected specified pluginpath in DNF configuration.'),
+            reporting.Summary(
+                'The option "pluginpath" is set in {}. The path of plugins are evolving between '
+                'system releases which can cause issues with the upgrade. '
+                .format(DNF_CONFIG_PATH)
+            ),
+            reporting.Remediation(
+                hint='Option pluginpath needs to be removed or commented out before running the upgrade. '
+                     'Use "sed -i "s/^pluginpath=.*/#pluginpath=/" /etc/dnf/dnf.conf" '
+                     'to comment out the option and remove the value.'
+            ),
+            reporting.Severity(reporting.Severity.HIGH),
+            reporting.Groups([reporting.Groups.INHIBITOR]),
+            reporting.Groups([reporting.Groups.REPOSITORY]),
```

Comment:
that's kind of incorrect as repository is more in the meaning of DNF repositories I guess. we do not have it defined the meaning of the group, but from other uses in the project, it's connected to DNF repositories. 
```suggestion
```